### ISSUE TITLE
Make mono repo containing `posit-dev/shinylive`, `posit-dev/r-shinylive`, `posit-dev/py-shinylive`

### ISSUE THREAD

**Issue by schloerke** [posit-dev/r-shinylive#3]
(Can probably exclude `quarto-ext/shinylive` as it is not a single version when released.)

Whereas `r-shinylive` and `py-shinylive` have a fixed version of `posit-dev/shinylive` that they work with. ... so they should all be in the same repo.

**Comment by nealrichardson** [comment]
I agree that a monorepo seems like it would make sense here--I almost suggested it when you asked for this repo but figured I'd just give you what you asked for :)

Would be curious if others have strong opinions here.

**Comment by wch** [comment]
I'm in favor of a monorepo. 👍

**Comment by nealrichardson** [comment]
Cool. I'm guessing we'd want to have shinylive be the monorepo and put py-shinylive and r-shinylive in subdirectories. Looks like we can do that with git and preserve the commit histories along with it.
